specs/WinRT/WinRTAPIContracts.md

```diff
@@ -0,0 +1,80 @@
+# Windows App SDK API Contracts
+
+## Summary
+Customers trust that Microsoft products are sound. As Windows App SDK evolves and more developers
+use Windows App SDK in their own projects Windows App SDK needs a way, at build time and run time,
+to allow apps to light up API features and to gracefully degrade. This can be achieved by looking
+at the deltas between APIs.
+
+In order to allow apps to depend on API versions Windows App SDK will add contracts to all their
+WinRT APIs.
+
+
+## Architecture Decisions
+1. Contract boundaries
+   1. One contract per feature/component/sub-system
+   2. Contracts will not span across a transport package/repo.
+3. Contract numbering.
```

Comment:
NIT: Inconsistent formatting

3+5 are "<number> Blah blah." whereas 1+4+6 are "<number> Blah blah" (no dot)

You're also doing formatting that Markdown doesn't respect.

```
4. Enforcement
The build system has...
```

which you'd think is 2 lines but Markdown (like HTML) is flow based. Take a look at the "rich diff" for the formatted document
![image](https://user-images.githubusercontent.com/72222/157755958-2d895316-c31f-4125-b3fe-a93c41eb5d18.png)
Very different from the raw layout.

Reply:
Interesting.  Many of your comments have been formatting related.  I've been using stackedit for md.

Reply:
seems like stackedit and github's viewers don't agree.